trouble_server/game/gameService.js

```diff
@@ -0,0 +1,75 @@
+const database = require('./model/database')
+const pieces = require('./model/piece')
+let Piece = pieces.Piece
+
+/**
+ * Represents a palyer data transfer object
+ * @constructor
+ * @param {string} name - user name
+ * @param {string} color - user color
+ */
+class playerDTO {
+    constructor (name, color) {
+        this.name = name
+        this.color = color
+    } 
+}
+
+
+/**
+ * Handle the dice rolling event
+ * @param {number} gameID - the gameID of current game
+ * @returns {number} the result of rolling dice 
+ */
+const handleDiceRoll = (gameID) => {
+    let currentGame = database.gameList[gameID]
+    let rollResult = currentGame.rollDice()
+    // socketManager.sendRollResult(rollResult)
```

Comment:
How come we're returning a result instead of calling sendRollResult? I don't think the dice roll event handler should directly emit another message. It smells of multiple responsibilities.